tests/integration/_cli2runner.py

```diff
@@ -24,9 +24,17 @@ class Cli2Runner:
         "run": "override in subclass",
     }
 
-    def run_test(self, mocked_runner, tmpdir, cli_entry, config_fixture, expected):
+    def run_test(self, mocked_runner, mocker, tmpdir, cli_entry, config_fixture, expected):
         # pylint: disable=no-self-use
-        """the function to run the test and assert"""
+        """Confirm execution of ``cli.main()`` produces the desired results.
+
+        :param mocked_runner: A patched instance of runner
+        :param mocker: The pytest-mock fixture
+        :param tmpdir: A fixture generating a unique temporary directory
+        :param cli_entry: The CLI entry to set as ``sys.argv``
```

Comment:
```suggestion
        :param cli_entry: The CLI entry to set as :data:`sys.argv`
```

(per https://webknjaz.github.io/intersphinx-untangled/docs.python.org/)